packages/blade/src/components/Input/_decisions/decisions.md

```diff
@@ -0,0 +1,295 @@
+| name                                                                                              | `string`                                                                                                                                                                                                                                               | No       |                                                           | The name of the checkbox group, [useful in form submissions](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#name)                                                                                                                                                    |
+| onChange                                                                                          | `({ event, name, value }) => {}`                                                                                                                                                                                                                       | No       |                                                           | Function called when the value of the input field changes                                                                                                                                                                                                                             |
+| onBlur                                                                                            | `({ event, name, value }) => {}`                                                                                                                                                                                                                       | No       |                                                           | Function called when the input field loses focus                                                                                                                                                                                                                                      |
+| isDisabled                                                                                        | `boolean`                                                                                                                                                                                                                                              | No       | `false`                                                   | Marks the input field as disabled. It'll also set `aria-disabled` to true and hence the input field will loose keyboard focus.                                                                                                                                                        |
```

Comment:
**Question:** Possible to have an alias to this properly -  `disabled` - since this would be in conjuncture to the original HTML API to <input/> element